.github/workflows/build.yml

```diff
@@ -0,0 +1,37 @@
+name: Test buildability
+
+on:
+  push:
+    branches: [ v2.2.6-for-tarantool-httpng ]
+  pull_request:
+    branches: [ v2.2.6-for-tarantool-httpng ]
+
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: ['ubuntu-latest', 'macos-latest']
+        build-type: ['Release', 'Debug']
+    runs-on: ${{matrix.os}}
+
+    steps:
+    - name: Where we are?
+      run: |
+        uname -a; cat /etc/os-release || cat /etc/issue || python -m platform || true
+        gcc --version || true
+        clang --version || true
+        ld --version || true
+      shell: bash
+
+    - uses: actions/checkout@v2
+
+      #    - name: Checkout submodules
+      #      run: git submodule update --init --recursive
```

Comment:
please remove it

Reply:
My mistake, I have overlooked that. Thank you.